content/posts/enable-beta-addons.md

```diff
@@ -0,0 +1,146 @@
+
+
+In the last year, we have been focusing ourselves on providing a better experience for authors and readers of the documentation.
+Today, we are thrilled to introduce our latest feature,
+designed on top of users' feedback to build a better experience for them in our platform: **Read the Docs Addons**.
+
+Read the Docs Addons helps authors to keep their documentation updated by focusing on the most relevant pages,
+while simplifying the review process --and also helps readers to navigate the documentation in a simple way,
+always knowing the versions they are reading avoiding confusions.
+
+Continue reading to explore the full list of addons and how to enable them in your project.
+
+## Full list of addons
+
+<!--
+TODO describe each of the addons into its own section,
+similar to what we have in https://github.com/readthedocs/addons/blob/main/README.md
+
+Consider also https://about.readthedocs.com/docs-as-code/
+-->
+
+### Analytics
+
+![Analytics Addons](/images/addons-analytics.png)
+
+Analytics allows you to explore your documentation traffic to understand what are the pages your readers read the most.
+With this data, your team is able to **prioritize the work on pages with high traffic** over others with less.
+Besides, you can find out what are the terms your users search the most and
+write documentation pages that include those terms so they can easily find them.
+
+### DocDiff
+
+![Docdiff Addons](/images/addons-docdiff.gif)
+
+DocDiff shows changes made on the rendered versions by **visually highlighing the differences between
+the current pull request and the latest version of the project's documentation**.
+This addons allows reviewers to quickly find what has changed on this pull request
+but also find style/rendering issues before accepting and merging the pull request.
+
+### Pull request notification
+
+![Pull request notification](/images/addons-pull-request-notification.png)
+
+This notification warns users letting them know the documentation page they are reading
+comes from a pull request to clearly distinguish this version from the official one.
+
+
+### Flyout
+
+![Flyout Addons](/images/addons-flyout.png)
+
+We call _flyout_ to the small floating Read the Docs section where all the documentation versions and translations are listed.
+This allows readers to **find the exact version they are looking for** and also read the documention on their own language if it's available.
+Besides, it contains useful links to go to the project's home, builds page and offline formats.
+
+### Non-stable notification
+
+![Non-stable notification](/images/addons-non-stable-notification.png)
+
+A notification on all _non-stable_ version is shown to clearly communicate readers
+they may be reading an outdated version of the documentation.
+This helps readers to avoid confusions and quickly jump into the _latest_ version of the documentation.
```

Comment:
```suggestion
This helps readers to avoid confusion and quickly jump into the _latest_ version of the documentation.
```

Also, did you mean "stable" here?

Reply:
```suggestion
This helps readers to avoid confusion and quickly jump into the _stable_ version of the documentation.
```

Reply:
He, yeah. Technically, the correct would be "latest stable version", but that adds more confusion I guess 😅 . Saying _stable_ is accurate as well.